### PR TITLE
for cancel an invoice

### PR DESCRIPTION
**Pull request by nuhamozaini**

### CODE REVIEW

**Comment by ecleel** [comment]
Great work! could you also update the dependencies to fix security issues on it 
https://github.com/moyasar/moyasar-node/network/alerts

**Comment by nuhamozaini** [comment]
I did, all dependencies are up to date now and according to `npm audit`, I found no security issues.

**Comment by ecleel** [comment]
pumped to version to 0.5.0

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "moyasar",
-  "version": "0.4.11",
+  "version": " 0.5.0",
```
Comment (ecleel) [inline package.json]: Remove whitespace

Comment (nuhamozaini) [reply package.json]: Done

**Pull request merged**